main.js

```diff
@@ -150,5 +151,5 @@ module.exports = function(options) {
 	return app;
 };
 
-module.exports.Router = express.Router
-
+module.exports.Router = express.Router;
+module.exports.services = serviceMatchers;
```

Comment:
oh just thought you might want to have the array in a separate repo and then pull it in where you need it
but i'm fine with exposing it like this